apps/participants/views.py

```diff
@@ -168,10 +168,10 @@ def cancel_schedule(request, team_slug: str, participant_id: int, schedule_id: s
 
 @permission_required("experiments.view_participant")
 @login_and_team_required
-def participant_identifiers_by_experiment(request, team_slug: str, experiment_id: int):
+def participants_data_by_experiment(request, team_slug: str, experiment_id: int):
```

Comment:
this name change and the url name changes don't seem necessary. The view still returns participant identifiers and not participant data (which is typically used to refer to ParticipantData)